src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -570,8 +570,8 @@
   <data name="TupleElementLocationCountMismatch" xml:space="preserve">
     <value>If tuple element locations are specified, the number of locations must match the cardinality of the tuple.</value>
   </data>
-  <data name="TuplesNeedAtLeastTwoElements" xml:space="preserve">
-    <value>Tuples must have at least two elements.</value>
+  <data name="TuplesNeedAtLeastOneElement" xml:space="preserve">
+    <value>Tuples must have at least one element.</value>
```

Comment:
>Tuples must have at least one element. [](start = 11, length = 38)

The error feels somewhat misleading. Tuples actually cannot have one element. However, tuple type symbol can. Consider using message "Tuple type symbol must have at least one element."